alfresco-transform-libreoffice/alfresco-transform-libreoffice/src/main/resources/libreoffice_engine_config.json

```diff
@@ -197,39 +197,39 @@
         {"sourceMediaType": "application/vnd.ms-excel.sheet.macroenabled.12",                                                                            "targetMediaType": "application/vnd.oasis.opendocument.spreadsheet-template" },
         {"sourceMediaType": "application/vnd.ms-excel.sheet.macroenabled.12",                                                                            "targetMediaType": "text/csv" },
         {"sourceMediaType": "application/vnd.ms-excel.sheet.macroenabled.12",                                                                            "targetMediaType": "text/tab-separated-values" },
         {"sourceMediaType": "application/vnd.ms-excel.sheet.macroenabled.12",                                                                            "targetMediaType": "application/vnd.ms-excel" },
-        {"sourceMediaType": "application/vnd.ms-excel.sheet.macroenabled.12",                            "maxSourceSizeBytes": 1572864,                  "targetMediaType": "application/pdf" },
+        {"sourceMediaType": "application/vnd.ms-excel.sheet.macroenabled.12",                            "maxSourceSizeBytes": 12582912,                 "targetMediaType": "application/pdf" },
 
         {"sourceMediaType": "application/vnd.openxmlformats-officedocument.spreadsheetml.sheet",                                                         "targetMediaType": "text/html" },
         {"sourceMediaType": "application/vnd.openxmlformats-officedocument.spreadsheetml.sheet",                                                         "targetMediaType": "application/vnd.oasis.opendocument.spreadsheet" },
         {"sourceMediaType": "application/vnd.openxmlformats-officedocument.spreadsheetml.sheet",                                                         "targetMediaType": "application/vnd.oasis.opendocument.spreadsheet-template" },
         {"sourceMediaType": "application/vnd.openxmlformats-officedocument.spreadsheetml.sheet",                                                         "targetMediaType": "text/csv" },
         {"sourceMediaType": "application/vnd.openxmlformats-officedocument.spreadsheetml.sheet",                                                         "targetMediaType": "text/tab-separated-values" },
         {"sourceMediaType": "application/vnd.openxmlformats-officedocument.spreadsheetml.sheet",                                                         "targetMediaType": "application/vnd.ms-excel" },
-        {"sourceMediaType": "application/vnd.openxmlformats-officedocument.spreadsheetml.sheet",         "maxSourceSizeBytes": 1572864,                  "targetMediaType": "application/pdf" },
+        {"sourceMediaType": "application/vnd.openxmlformats-officedocument.spreadsheetml.sheet",         "maxSourceSizeBytes": 10485760,                 "targetMediaType": "application/pdf" },
```

Comment:
This one is 15728640 in T-Service

Reply:
Great catch - it's been a long few days. Fixing now.

Reply:
Fixed.